db/seeds.rb

```diff
@@ -41,7 +41,7 @@ def time_floor(time, seconds = 15.minutes)
   campus = Campus.all.to_a.sample
   10.times do
     user = User.all.to_a.select{|u| u.campus.id == campus.id}.sample
-    room = campus.rooms.new(name:Faker::Name.name,location:Faker::Address.city, capacity: rand(10..30), picture_url: Faker::Avatar.image, events_count:0)
+    room = campus.rooms.new(name:Faker::Name.name,location:Faker::Address.city, capacity: rand(10..30), picture_url: Faker::Avatar.image, events_count:0, average_capacity_use: rand(1..100))
```

Comment:
Kind of unrelated to this pull request, but worth noting anyway, if you break up long one liners like this into multiple lines, it slightly cuts down on the chance of a merge conflict if multiple people have to edit the same file.

```rb
room = campus.rooms.new(name: Faker::Name.name,
                          location: Faker::Address.city, 
                          capacity: rand(10..30), 
                          picture_url: Faker::Avatar.image, 
                          events_count:0, 
                          average_capacity_use: rand(1..100))
 ```